src/passkit.cs

```diff
@@ -738,7 +800,7 @@ interface PKPass : NSSecureCoding, NSCopying {
 		[Export ("localizedValueForFieldKey:")]
 		NSObject GetLocalizedValue (NSString key); // TODO: Should be enum for PKPassLibraryUserInfoKey
 
-#if !XAMCORE_4_0
+#if !XAMCORE_4_0 || !MONOMAC
```

Comment:
We're not defining XAMCORE_4_0, so adding `|| !MONOMAC` doesn't change anything. Do you mean `&& !MONOMAC`?